example/v2/example.dart

```diff
@@ -0,0 +1,54 @@
+//import 'dart:convert';
```

Comment:
Let's not use commented out code blocks, even in examples. If a structure is needed, let's refactor it into methods. (same for example/v3/example.dart)

Reply:
removed commented codes